xml/depl_nodes.xml

```diff
@@ -5002,6 +5008,461 @@ getent group postgres | cut -d: -f3</screen>
 
   </sect1>
 
+    <sect1 xml:id="sec.depl.ostack.monasca">
+    <title>Deploying &o_monitor;</title>
+    <para>
+      &o_monitor; is an open-source monitoring-as-a-service solution that integrates
+      with OpenStack. &o_monitor; is designed for scalability, high performance, and
+      fault tolerance.
+    </para>
+    <para>
+      Since not all &o_monitor; settings are exposed in the &barcl;
+      graphical interface, it is recommended to configure &o_monitor; through the the
+      <guimenu>Raw</guimenu> interface. Below are the options that can be configured via the <guimenu>Raw</guimenu> interface of the &o_monitor; &barcl;.
+    </para>
+
+  <figure>
+   <title>The &o_monitor; &barcl; Raw Mode</title>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="depl_barclamp_monasca_raw.png" width="100%" format="png"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="depl_barclamp_monasca_raw.png" width="75%" format="png"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+
+    <bridgehead renderas="sect3"><guimenu>agent: settings for openstack-monasca-agent</guimenu></bridgehead>
+
+    <variablelist>
+      <varlistentry>
+      <term>keystone</term>
+      <listitem>
+        <para>
+          Contains &o_ident; credentials that the agents use to send
+	  metrics. Do not change the this options, as it is configured by &crow;.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>insecure</term>
+      <listitem>
+        <para>
+          Specifies whether SSL certificates are verified when
+	  communicating with &o_ident;. If set to <literal>false</literal>, the
+	  <literal>ca_file</literal> option must be disabled as well.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>ca_file</term>
+      <listitem>
+        <para>
+          Specifies whether a CA certificate is used for verifying &o_ident;'s SSL certificate.
```

Comment:
This doesn't specify whether to use a CA certificate for verification (that is what the `insecure` setting is for) but the filesystem location of such a CA file.